pkg/tfgen/generate.go

```diff
@@ -370,7 +370,7 @@ func (g *generator) Generate() error {
 	}
 
 	// Print out some documentation stats as a summary afterwards.
-	printDocStats(false, false)
+	printDocStats(true, true)
```

Comment:
Do we want this debugging info by default now?

Reply:
Oops, unintentional! Will fix